tools/python/maps_generator/__main__.py

```diff
@@ -1,22 +1,55 @@
 import logging
 import os
-from argparse import ArgumentParser
+from argparse import ArgumentParser, RawDescriptionHelpFormatter
 
 from .generator import settings
 from .generator.env import Env, find_last_build_dir, WORLDS_NAMES
 from .generator.exceptions import ContinueError, SkipError, ValidationError
 from .maps_generator import (generate_maps, generate_coasts, reset_to_stage,
                              ALL_STAGES, stage_download_production_external,
                              stage_descriptions, stage_ugc, stage_popularity,
-                             stage_localads, stages_as_string)
+                             stage_localads, stage_statistics,
+                             stages_as_string)
 from .utils.collections import unique
 
 logger = logging.getLogger("maps_generator")
 
 
+examples = """Examples:
+1) Non-standard planet
+    If I want to generate maps for Japan I must complete the following steps:
+    1. Open https://download.geofabrik.de/asia/japan.html and copy url of osm.pbf
+     and md5sum files.
+    2. Edit ini file:
+    maps_generator$ vim var/etc/map_generator.ini
+
+    ...
+    [Main]
+    ...
+    DEBUG: 0
+    ...
+    [External]
+    PLANET_URL: https://download.geofabrik.de/asia/japan-latest.osm.pbf
+    PLANET_MD5_URL: https://download.geofabrik.de/asia/japan-latest.osm.pbf.md5
+    ...
+    
+    3. Run
+    python$ python3.6 -m maps_generator --countries="World, WorldCoasts, Japan_*" --skip="update_planet"
+
+    We must skip the step of updating the planet, because it is a non-standard planet.
+2) Rebuild stages:
+    For example, you changed routing code in omim project and want to regenerate maps.
```

Comment:
Please use one pronoun throughout: currently, you are using "I", "We" and "You".

Reply:
++ you